src/pages/EnablePayments/FeesAndTerms/substeps/TermsStep.tsx

```diff
@@ -28,18 +25,21 @@ function HaveReadAndAgreeLabel() {
 
 function AgreeToTheLabel() {
     const {translate} = useLocalize();
+    const [userWallet] = useOnyx(ONYXKEYS.USER_WALLET);
+
+    const walletAgreementUrl = userWallet?.walletProgramID === CONST.WALLET.MTL_WALLET_PROGRAM_ID ? CONST.WALLET_AGREEMENT_URL : CONST.BANCORP_WALLET_AGREEMENT_URL;
```

Comment:
If `userWallet` is null, this will default to `BANCORP_WALLET_AGREEMENT_URL` right? We should flip that so our MTL agreement is the default since that will be the default program moving forward. The bancorp wallet program ID is `'660'` - I don't see it in `src/CONST.ts` currently